docs/root/intro/arch_overview/upstream/load_reporting_service.rst

```diff
@@ -12,4 +12,4 @@ to a node it is interested in getting the load reports for. Envoy in this node w
 :ref:`LoadStatsRequest <envoy_v3_api_msg_service.load_stats.v3.LoadStatsRequest>`. This is done periodically
 based on the :ref:`load reporting interval <envoy_v3_api_field_service.load_stats.v3.LoadStatsResponse.load_reporting_interval>`
 
-Envoy config with LRS can be found at :repo:`/examples/load-reporting-service/service-envoy-w-lrs.yaml`.
+Envoy config with LRS can be found at :repo:`/examples/load-reporting-service/envoy.yaml`.
```

Comment:
@adisuissa it would be better to use the artefact that is available in the example docs

which i believe is ...

```suggestion
Envoy config with LRS can be found at `envoy.yml <_include/load-reporting-service/envoy.yml>`
```

Reply:
Sure.
Just to double check, the suffix should be "yaml", right?
https://github.com/envoyproxy/envoy/blob/main/examples/load-reporting-service/envoy.yaml

Reply:
yeah - it copies the file from `examples/` dir into docs rst/build

```console
$ ls examples/load-reporting-service/
docker-compose.yaml  envoy.yaml  example.rst  go.mod  go.sum  main.go  README.md  send_requests.sh  server  verify.sh
```

Reply:
ah - apologies - i think this should have been ...

```suggestion
Envoy config with LRS can be found at `envoy.yml </_include/load-reporting-service/envoy.yml>`
```

Reply:
note: preceeding `/`

Reply:
Sure, trying that.
FWIW, I don't see the "/include" in places such as [this](https://github.com/envoyproxy/envoy/blob/dd7d9b0326967e32aef6accd87b30bf7b9543f53/docs/root/start/quick-start/configuration-static.rst?plain=1#L14).
That said, it may be because they are all `:download:` and not `:repo:`. Testing it out.

Reply:
relative links - as this is not in the examples rst it needs to use an absolute link

Reply:
I'm not seeing a link, just text. Is this expected?
https://storage.googleapis.com/envoy-pr/4f60c1a/docs/intro/arch_overview/upstream/load_reporting_service.html#load-reporting-service-lrs

Reply:
apologies - no - it should have `:ref:` at the beginning - apologies for any confusion

Reply:
my reason for pushing back/suggesting change is that it ensures the docs will always work (ie uses its own artefacts) whereas if we change the layout of repo in future (something im ~currently working on) the repo links will break

Reply:
build is broken with '/', so trying without.

Reply:
pretty sure it needs it - if i get some time tomorrow ill look at the rst file tree and figure out what it should be

Reply:
k @adisuissa built the rst and from that i can see the link should be `/start/sandboxes/_include/load-reporting-service/envoy.yaml`